sdk/servicebus/service-bus/src/serializers/topicResourceSerializer.ts

```diff
@@ -138,11 +138,11 @@ export interface TopicOptions {
   sizeInBytes?: number;
 
   /**
-   * Specifies the maximum topic size in megabytes. Any attempt to enqueue a message 
```

Comment:
I wonder that since this is about the message body size and service way of reporting the sizes, we needn't worry about it so long as the send operations in AMQP talk about it?

Reply:
Discussed offline, we agreed that bytes on these properties are accurate. And that we might want to elaborate more on the send API or others as necessary - to help users understand how the JS object we construct translates to expected byte size.
Filed issue https://github.com/Azure/azure-sdk-for-js/issues/6683 so we remember to check this.